server/src/database.js

```diff
@@ -64,12 +96,80 @@ const createStore = () => {
     title: Sequelize.STRING,
     description: Sequelize.STRING,
     price: Sequelize.FLOAT,
-    maxParticipants: Sequelize.INTEGER,
+    type: Sequelize.INTEGER,
+    // 'type' specifies which type of the event is.
+    // Enum type is not SQL-standard and it is hard to add a new enum value.
+    // If we want to, we have to use ALTER TABLE statement.
+    // So we define type as INTEGER.
+    // 0: BookClub
+    thumbnailUrl: Sequelize.STRING,
   }, {
     sequelize,
     modelName: 'event',
   });
 
+  EventBookClub.init({
+    eventId: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      references: {
+        model: Event,
+        key: 'id',
+      },
+    },
+    bookTitle: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    author: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: Sequelize.STRING,
+    },
+    ISBN: {
+      type: Sequelize.INTEGER,
+    },
+  }, {
+    sequelize,
+    modelName: 'eventBookClub',
+    timestamps: false,
+  });
+
+  Review.init({
+    userId: {
+      type: Sequelize.INTEGER,
+      primaryKey: true,
+      references: {
+        model: User,
+        key: 'id',
+      },
+    },
+    eventId: {
```

Comment:
This is for future thinking, and we can add TODOs to fix later.

Is a Review tied to an Event? Should that be tied to Schedule instead? Or should that be tied to the host ID? Some questions:
- Does the product ever need to indicate which schedule the user participated (i.e. "This is a review about the event held on June 15, 2019")?
- Can the host "delete" event or schedule? If that happens, where do all the reviews go? Reviews should always stay in the system (never deleted). 
- Take a look at how AirBnB handles reviews. There, all reviews are tied to the hosts regardless of different houses she has, or regardless of different bookings each house has. But that's how it's shown on UI - how can you implement this on the backend? Should reviews be tied to all 3 (hostID, eventID, scheduleID), and display the appropriate info at different product request?